generators/generators-commons/src/main/java/uk/gov/justice/services/generators/commons/helper/PathToUrlResolver.java

```diff
@@ -12,7 +12,7 @@ public URL resolveToUrl(final Path baseDir, final Path path) {
         try {
             return baseDir.resolve(path).toUri().toURL();
         } catch (final MalformedURLException e) {
-            throw new FileParserException(format("Cannot resolve path as URL %s", path), e);
+            throw new FileParserException(format("Cannot resolve path as URL %s", path.toFile()), e);
```

Comment:
If it OK to just have the path instead?
i.e. if path is null, you get NPE.

Reply:
PATH has no to string so returns hashcode